string name
float64 area
float64 angle_horizontal
float64 angle_vertical
